### ISSUE TITLE
New NVIDIA Driver 455.23.04 caused Vulkan Shader issues on Steam, had to roll back via timeshift.

### ISSUE THREAD

**Issue by Redbatman89** [pop-os/pop#1275]
So a few hours ago I had downloaded the latest NVIDIA driver only after restart I noticed that the background processing of vulkan shaders was taking quite a long time. Rise of Tomb raider was taking almost 20mins so I rolled back to 440.100.

I don't know why it happened but now my shaders are fast again and are back to usual processing time. Please POP OS take another look at this driver, it's odd the driver was pushed out since it's a beta driver for the RTX 3000 series.

**Comment by NinjaTurtle007** [comment]
This driver messed up my setup as well and the thing is you can't revert back
sudo apt-get purge '*nvidia*'
sudo apt install nvidia-driver-440

it still installs 455!
and I don't have timeshift setup...

**Comment by NinjaTurtle007** [comment]
@leviport  yes im on 455.28 and still have the issue, it's decribed in more detail here:
https://github.com/pop-os/pop/issues/1289#issuecomment-707796209

@Redbatman89  how did you revert to 440?
When I tried (purge *nvidia, apt install nvidia-driver-440) it still installed 455.28 
i was even unable to boot at one point
